vendors/pc/boards/windows/aws_demos/application_code/runtimeTimer.c

```diff
@@ -0,0 +1,70 @@
+#include "runtimeTimer.h"
+
+#include "FreeRTOS.h"
+#include "FreeRTOSIPConfig.h"
+#include "task.h"
+#include "timers.h"
+
+#define MS_PER_MINUTE (SECONDS_PER_MINUTE * MS_PER_SECOND)
+#define SECONDS_PER_MINUTE (60)
+#define MS_PER_SECOND (1000)
+
+#define RUNTIME_TASK_INTERRUPT_NUMBER (3)
+
+volatile unsigned long long runtime_ms = 0;
+static StaticTimer_t xRuntimeTimerBuffer[configMINIMAL_STACK_SIZE];
+static unsigned long runtimeTimerId = 1004; // Arbitrary value
+// portTICK_PERIOD_MS is 1. Windows Simulator Tick is 1kHZ.
+const TickType_t xDelay = 1 / portTICK_PERIOD_MS; 
```

Comment:
Changing xDelay to values from 1 to 1000 did not affect the accuracy of the calculated run time. The calculation is correct though, because I use the same calculation for the Windows System Ticks time, and that yields a more correct solution.